packages/react/src/stories/ic-text-field.stories.mdx

```diff
@@ -482,3 +482,207 @@ export const Uncontrolled = () => {
+  validationText: "",
+  debounce: 0
+};
+
+<Canvas>
+  <Story
+    args={defaultArgs}
+    argTypes={{
+      autocapitalize: {
+        options: ["off", "none", "on", "sentences", "words", "characters"],
+        control: { type: "select" }
+      },
+      autocomplete: {
+        options: [
+          "on",
+          "off",
+          "name",
+          "honorific-prefix",
+          "given-name",
+          "additional-name",
+          "family-name",
+          "honorific-suffix",
+          "nickname",
+          "email",
+          "username",
+          "new-password",
+          "current-password",
+          "one-time-code",
+          "organization-title",
+          "organization",
+          "street-address",
+          "address-line1",
+          "address-line2",
+          "address-line3",
+          "address-level4",
+          "address-level3",
+          "address-level2",
+          "address-level1",
+          "country",
+          "country-name",
+          "postal-code",
+          "cc-name",
+          "cc-given-name",
+          "cc-additional-name",
+          "cc-family-name",
+          "cc-family-name",
+          "cc-number",
+          "cc-exp",
+          "cc-exp-month",
+          "cc-exp-year",
+          "cc-csc",
+          "cc-type",
+          "transaction-currency",
+          "transaction-amount",
+          "language",
+          "bday",
+          "bday-day",
+          "bday-month",
+          "bday-year",
+          "sex",
+          "tel",
+          "tel-country-code",
+          "tel-national",
+          "tel-area-code",
+          "tel-local",
+          "tel-extension",
+          "impp",
+          "url",
+          "photo"
+        ],
+        control: { type: "select" }
+      },
+      autocorrect: {
+        options: ["off", "on"],
+        control: { type: "inline-radio" }
+      },
+      autoFocus: {
+        control: { type: "boolean" }
+      },
+      disabled: {
+        control: { type: "boolean" }
+      },
+      fullWidth: {
+        control: { type: "boolean" }
+      },
+      hideLabel: {
+        control: { type: "boolean" }
+      },
+      inputmode: {
+        options: ["none", "text", "tel", "url", "email", "numeric", "decimal", "search"],
+        control: { type: "select" }
+      },
+      readonly: {
+        control: { type: "boolean" }
+      },
+      required: {
+        control: { type: "boolean" }
+      },
+      resize: {
+        control: { type: "boolean" }
+      },
+      size: {
+        options: ["default", "small"],
+        control: { type: "inline-radio" }
+      },
+      spellcheck: {
+        control: { type: "boolean" }
+      },
+      type: {
+        options: ["email", "password", "tel", "text", "url", "number", "search"],
+        control: { type: "select" }
+      },
+      validationInline: {
+        control: { type: "boolean" }
+      },
+      validationStatus: {
+        options: ["warning", "error", "success", ""],
```

Comment:
Storybook allows you to [map options to other values](https://storybook.js.org/docs/api/arg-types#mapping). Please can you use this feature instead of having an empty string as one of the options in the dropdown

It has been used here really well:
https://github.com/mi6/ic-ui-kit/blob/72c40886186cecd33723470127614a9152b77710/packages/web-components/src/components/ic-card/ic-card.stories.mdx?plain=1#L865-L900